tests/fixtures/renderer_test_user_rules/test_rule_1.yaml

```diff
@@ -0,0 +1,22 @@
+es_host: 127.0.0.1
+es_port: 80
+index: test_index
+type: any
+alert:
+  - email:
+      email: example@gmail.com
+      email_config_name: mail_1
+  - email:
+      email: test@gmail.com
+      email_config_name: mail_2
+  - email:
+      email: test2@gmail.com
+      email_config_name: default
+  - slack:
+      slack_username_override: test_user
```

Comment:
Why override? Let's use just `username` and `channel`
Also, can I use only `channel` or only `username`?
This is user interface it should be simpler

Reply:
I used standard ElastAlert option names. Should I add custom options and implement the generation of standard ElastAlert options?

Reply:
Let's add options mapping from user interface to rules file

Reply:
You mean adding appropriate options to jinja template to render them in EA rule files?